setup.sh

```diff
@@ -105,6 +105,12 @@ echo "Setting up TnP tools..."
 #git cms-addpkg DataFormats/RecoCandidate
 #git cms-addpkg PhysiscsTools/TagAndProbe
 git cms-merge-topic -u matteosan1:egm_tnp_76X_v1
+git cms-merge-topic -u matteosan1:smearer_76X_test
+git remote add cmssw-ferriff https://github.com/ferriff/cmssw.git
+git fetch cmssw-ferriff
```

Comment:
On Tue, Feb 23, 2016 at 01:40:07AM -0800, musella wrote:
> > @@ -105,6 +105,12 @@ echo "Setting up TnP tools..."
> >  #git cms-addpkg DataFormats/RecoCandidate
> >  #git cms-addpkg PhysiscsTools/TagAndProbe
> >  git cms-merge-topic -u matteosan1:egm_tnp_76X_v1
> > +git cms-merge-topic -u matteosan1:smearer_76X_test
> > +git remote add cmssw-ferriff https://github.com/ferriff/cmssw.git
> > +git fetch cmssw-ferriff
> 
> Could this cherry picks be put into a CMSSW topic branch?

They have been included yesterday by @matteosan1 in
matteosan1:smearer_76X_test , so after his confirmation those lines can
be removed by the setup.sh .